tests/unit/dialog/core.js

```diff
@@ -84,6 +84,39 @@ QUnit.test( "ARIA", function( assert ) {
 	element.remove();
 } );
 
+QUnit.test( "aria-modal", function( assert ) {
+	assert.expect( 9 );
+
+var element, wrapper;
```

Comment:
Oops, I messed up indentation:
```suggestion
	var element, wrapper;
```